flink-table/README.md

```diff
@@ -65,7 +66,8 @@ If you want to use Table API & SQL, check out the [documentation](https://nightl
 
 ### Notes
 
-No module except `flink-table-planner` should depend on `flink-table-runtime` in production classpath, 
+No module except `flink-table-planner` should depend on `flink-table-runtime` in production classpath,
```

Comment:
@slinkydeveloper Have you considered enforcing this via the enforcer plugin, similarly to what we do for [flink-rpc-akka](https://github.com/apache/flink/blob/master/pom.xml#L1730)?

Reply:
I didn't knew about that, but yep it probably makes sense to do so

Reply:
Is it possible to do so but allowing flink-table-planner for testing purpose?

Reply:
Gotcha, opened an issue here to track it https://issues.apache.org/jira/browse/FLINK-26054